front/chat/src/Components/Chat.svelte

```diff
@@ -195,6 +200,16 @@
                     />
                 {/if}
 
+                <ChatForumRooms
+                    {showForums}
+                    searchValue={searchValue.toLocaleLowerCase()}
+                    on:activeThread={handleActiveThread}
+                    on:showForums={handleShowForums}
```

Comment:
Why don't use this directly in the component `ChatForumRooms` and have a handler and props?